docs/data/charts/funnel/funnel.md

```diff
@@ -1,15 +1,111 @@
 ---
 title: React Funnel chart
 productId: x-charts
+components: FunnelChart, FunnelPlot
 ---
 
-# Charts - Funnel [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')🚧
+# Charts - Funnel [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
 
-<p class="description">Funnel charts allow to express quantity evolution along a process, such as audience engagement.</p>
+<p class="description">Funnel charts allow to express quantity evolution along a process, such as audience engagement, population education levels or yields of multiple processes.</p>
 
-:::warning
-The Funnel Chart component isn't available yet, but you can upvote [**this GitHub issue**](https://github.com/mui/mui-x/issues/7929) to see it arrive sooner.
+## Basics
 
-Don't hesitate to leave a comment there to influence what gets built.
-Especially if you already have a use case for this component, or if you're facing a pain point with your current solution.
-:::
+The funnel accepts a series which must have a data property containing an array of objects. Those objects should contain a property `value`. They can also have other optional properties, like `label` and `id`.
+
+{{"demo": "BasicFunnel.js"}}
+
+### Display legends
+
+The funnel chart displays a legend by default. The only requirement is to provide a `label` value in the data objects.
+
+To disable the legend, set the `hideLegend` property to `true`.
+
+{{"demo": "FunnelLegend.js"}}
+
+## Labels
+
+The funnel chart displays labels by default. It shows the `value` of the data item in each section.
+To format the labels, a `valueFormatter` function can be provided.
+
+{{"demo": "FunnelLabels.js"}}
```

Comment:
INstead of showing the dataIndex of the item, it might be more interesting to adapt the formatting according to the difference between tooltip and items

Reply:
Series value formatter doesn't have the tooltip/axis context.